provision/initramfs/init

```diff
@@ -351,7 +351,7 @@ if [ -n "$WWPOSTSHELL" -a "$WWPOSTSHELL" != "0" ]; then
     setsid /bin/cttyhack /bin/sh
 fi
 
-if [ -x "$NEWROOT/sbin/init" ]; then
+if [ -x "$NEWROOT/sbin/init" ] || [ -h "$NEWROOT/sbin/init" -a -x "$NEWROOT`readlink $NEWROOT/sbin/init`" ]; then
```

Comment:
We could probably just trust that if `$NEWROOT/sbin/init` is either executable or a symlink we are good. e.g.:

`if [ -x "$NEWROOT/sbin/init" -o -L "$NEWROOT/sbin/init" ]; then`

Thoughts?